docs/src/app_dev/app_val_tokenomics/feat_app_act_marker_nomics.rst

```diff
@@ -0,0 +1,83 @@
+
+   1. Find the fully qualified interface ID for
+   `FeaturedAppRight <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#type-splice-api-featuredapprightv1-featuredappright-34177>`__
+   which is
+   ``7804375fe5e4c6d5afe067bd314c42fe0b7d005a1300019c73154dd939da4dda:Splice.Api.FeaturedAppRightV1:FeaturedAppRight``
+   for ``Splice.Api.FeaturedAppRightV1``. The command ``daml damlc
+   inspect-dar`` can be used to find this.
+
+   2. Query the ledger using this ID to retrieve contracts from the Daml
+   ledger that implement the
+   `FeaturedAppRight <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#type-splice-api-featuredapprightv1-featuredappright-34177>`__
+   interface. See using
+   `toInterface <https://docs.digitalasset.com/build/3.3/reference/daml/interfaces.html#tointerface>`__
+   or
+   `toInterfaceContractId <https://docs.digitalasset.com/build/3.3/reference/daml/interfaces.html#tointerfacecontractid>`__
+   for more information.
+
+   3. In the application, using the ``FeaturedAppRight`` interface, exercise
+   the ``FeaturedAppRight_CreateActivityMarker`` choice. Set the
+   templateId to the fully qualified interface ID above.
+
+   4. For testing examples, please review the example DamlScript test
+   `here <https://github.com/hyperledger-labs/splice/blob/main/daml/splice-dso-governance-test/daml/Splice/Scripts/TestFeaturedAppActivityMarkers.daml>`__.
+
+A non-featured app cannot accrue a ``FeaturedAppActivityMarker``.
+
+Consider a single, simple transaction of a RWA which creates a single
+``FeaturedAppActivityMarker`` activity record for one ``provider`` and the
+``beneficiary`` is the ``provider``:
+
+      1. A `FeaturedAppActivityMarker <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-featuredappactivitymarker-16451>`__
+         contract is created in the business transaction. The ``provider`` is
+         set to the featured application provider's party. The ``beneficiary``
+         must be set (unlike the
+         `AppRewardCoupon <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-apprewardcoupon-57229>`__)
+         to the party that would receive the CC. The ``provider`` field of the
+         `FeaturedAppActivityMarker <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-featuredappactivitymarker-16451>`__
+         is set by calling the interface choice
+         `FeaturedAppRight_CreateActivityMarker <https://docs.dev.sync.global/app_dev/api/splice-api-featured-app-v1/Splice-Api-FeaturedAppRightV1.html#type-splice-api-featuredapprightv1-featuredapprightcreateactivitymarker-36646>`__.
+
+      2. No ``ValidatorRewardCoupon`` is created.
+
+There can be several ``FeaturedAppActivityMarkers`` per transaction tree
+which increases the total reward. It is also possible for a single
```

Comment:
Missing: that this is only allowed for composed transactions like a settlement transaction where trading venue and all the registries of the transferred assets would get featured app rewards. 

This is not enforced in code. However, the tokenomics committee enforces fair usage of one's featured app right.

Reply:
Added. 

Done.